src/colors.scss

```diff
@@ -1,2 +1 @@
-$white: #fff;
 $black: #000;
```

Comment:
Off-topic: I added this file as I found colors that were not from paragon. If all colors will be paragon and not set via CSS, if it's just black, we can probably get rid of this again later.